catalyst-gateway/tests/schema-mismatch/Earthfile

```diff
@@ -0,0 +1,34 @@
+VERSION --global-cache 0.7
+
+builder:
+    FROM github.com/input-output-hk/catalyst-ci/earthly/python:v2.7.0+python-base
+
+    COPY --dir ./schema_mismatch README.md .
```

Comment:
Do we really need to `COPY` README.md file here ?

Reply:
Yes, CI complains if no README is found in the package.